pkg/controller/pagerdutyintegration/clusterdeployment_deleted.go

```diff
@@ -57,7 +58,7 @@ func (r *ReconcilePagerDutyIntegration) handleDelete(pdclient pd.Client, pdi *pa
 		return nil
 	}
 
-	ClusterID := cd.Spec.ClusterName
+	clusterNS := strings.Split(cd.Namespace, "-")
```

Comment:
this is done twice so far. Can we move this to a util package or something that can be re-used?